<a href="https://colab.research.google.com/github/GijooHong/basic_python221028/blob/main/basic_python221028.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

1.사용자로부터 두 개의 정수를 받은 정수의 합, 정수의 차, 정수의 곱, 정수의 평균, 큰 수, 작은 수를 계산하여 화면에 출력하는 프로그랩을 작성하라. 파이썬이 제공하는 내 장 함수 max(x, y), min(x, y)을 사용해보자.

*   항목 추가
*   항목 추가



In [3]:
x = int(input("x의 값은?: "))
y = int(input("y의 값은?: "))

print("두 수의 합은 :", x + y)
print("두 수의 차는 :", x - y)
print("두 수의 곱은 :", x * y)
print("두 수의 평균은 :", (x + y)/2)
print("큰 수는? :", max(x, y))
print("작은 수는? :", min(x, y))

x의 값은?: 9
y의 값은?: 6
두 수의 합은 : 15
두 수의 차는 : 3
두 수의 곱은 : 54
두 수의 평균은 : 7.5
큰 수는? : 9
작은 수는? : 6


2.원기둥의 부피를 계산하는 프로그램 을 작성해보자. 원기둥의 부피는 다음과 같이 계산 한다.(계산식은 3.141592 r**2 h)

In [4]:
r =float(input("r값은? :"))
h =float(input("h값은? :"))

vol = 3.141592 * r**2 * h
print("원기둥의 부피:", vol)

r값은? :10
h값은? :10
원기둥의 부피: 3141.592


In [6]:
number = int(input("정수를 입력하시오.: "))

sum = 0
sum = sum + number % 10
number = number // 10

sum = sum + number % 10
number = number // 10

sum = sum + number % 10
number = number // 10

sum = sum + number % 10
number = number // 10

print("자리수의 합은: ", str(sum) )

정수를 입력하시오.: 1234
자리수의 합은:  10


In [8]:
x1 = int(input("x1값을 입력하시오.: "))
x2 = int(input("x2값을 입력하시오.: "))
y1 = int(input("y1값을 입력하시오.: "))
y2 = int(input("y2값을 입력하시오.: "))
dist = ((x2 - x1)**2 + (y2 - y1)**2)** 0.5
print("두점의 사이의 거리는? ", dist, "입니다.")

x1값을 입력하시오.: 0
x2값을 입력하시오.: 100
y1값을 입력하시오.: 0
y2값을 입력하시오.: 100
두점의 사이의 거리는?  141.4213562373095 입니다.


import turtle 
t = turtle.Turtle()
t.shape("turtle")
t.left(45)
t.forward(141)
t.setheading(0)
t.goto(0,0)
t.fd(100)
t.lt(90)
t.fd(100)